schemas/expconf/v0/log-pattern-policies.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://determined.ai/schemas/expconf/v0/log-pattern-policies.json",
+    "title": "LogPatternPoliciesConfig",
+    "type": "array",
+    "items": {
+        "$ref": "http://determined.ai/schemas/expconf/v0/log-pattern-policy.json"
```

Comment:
I followed the precedent of ```proxy-ports.json``` and ```proxy-port.json``` here.

Reply:
I spent a while talking about this with Ryan. If this was best practices, I wanted to understand why. He pointed out that (he thinks) some types *need* to be referred to in a separate document in order to be merged properly. I expect this was done once or twice and people have cargo-culted array elements ever since.

Ryan tested out inlining `log-policy.json` while I watched, and found it to work.

He also pointed me at schema's `make check`, which does a pretty good job linting the expconf directory, and would catch an error like if the inlining didn't work. 

It's not a big deal. Do whatever you think is best. Just relaying that you've got the option.